src/main/java/com/google/swarm/tokenization/avro/AvroReaderSplittableDoFn.java

```diff
@@ -82,8 +82,9 @@ public void processElement(ProcessContext c, RestrictionTracker<OffsetRange, Lon
         GenericRecord record = fileReader.next();
 
         // Output the Avro record
-        String outputKey = String.format("%s~%d", fileName, new Random().nextInt(keyRange));
-        c.outputWithTimestamp(KV.of(outputKey, record), Instant.now());
+        // Removed outputKey computation as it was unnecessary 
+        // and creating error logs in the pipeline
```

Comment:
Please remove this comment. Folks interested in understanding why can understand it from the the file history.
LGTM otherwise.
Thanks!

Reply:
done